kalite/updates/management/commands/videoscan.py

```diff
@@ -1,108 +0,0 @@
-"""
```

Comment:
> "import content" button
Is this a "rescan for content already in the content folder" or "load content from elsewhere into the content folder"? The latter is slated for Kolibri, the former would probably be good to have in KA Lite, if we're taking it out of startup.